src/electionguard/election_polynomial.py

```diff
@@ -81,11 +81,11 @@ def compute_polynomial_coordinate(
     :return: Polynomial used to share election keys
     """
 
-    exponent_modifier = ElementModQ(exponent_modifier)
+    exponent_modifier_q = ElementModQ(exponent_modifier)
```

Comment:
This is a good refactor, but should this be in this PR? Did this break anything?